doc/tutorial/quick-start.md

```diff
@@ -27,11 +27,11 @@
 
 ### Run the image build
 
-- Run 
+- Choose the OpenStack <cloud-name> from the clouds.yaml file, and set the desired image name as <image-name>.
 ```
-CLOUD_NAME=<OpenStack cloud name from clouds.yaml>
-IMAGE_NAME=<your desired image name>
-github-runner-image-builder run <cloud-name> <image-name>
+CLOUD_NAME=<cloud-name>
+IMAGE_NAME=<image-name>
+github-runner-image-builder run $CLOUD_NAME $IMAGE_NAME
 ```
 to start building the image.
```

Comment:
I think it would be better if this was a complete sentence, perhaps "This event begins building the image."?

Reply:
One other nitpick: Is there a way for the user to check that everything is running or that the images have been successfully built?